static/js/apps/eval_retrieval_generation/overall_feedback.tsx

```diff
@@ -45,80 +46,118 @@ const LOADING_CONTAINER_ID = "form-container";
 // object to hold information about a question
 interface QuestionConfig {
   title: string;
-  question: string;
-  responseOptions: Record<string, string>;
+  questions: {
+    firestoreKey: string;
+    question: string;
+    responseOptions: Record<string, string>;
+  }[];
 }
 
 // Dictionary of feedback stage -> eval type -> question config
 const QUESTION_CONFIG: Record<string, Record<string, QuestionConfig>> = {
   [FeedbackStage.OVERALL_QUESTIONS]: {
     [EvalType.RAG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_QUESTIONS_KEY,
+          question:
+            "Do the generated questions seem sufficient to answer the query?",
+          responseOptions: {
+            [OVERALL_QUESTIONS_OPTION_HAS_MISSING]: "Missing obvious questions",
+            [OVERALL_QUESTIONS_OPTION_NONE_MISSING]:
+              "No obvious questions missing",
+          },
+        },
+      ],
       title: "QUESTIONS EVALUATION",
-      question:
-        "Do the generated questions seem sufficient to answer the query?",
-      responseOptions: {
-        [OVERALL_QUESTIONS_OPTION_HAS_MISSING]: "Missing obvious questions",
-        [OVERALL_QUESTIONS_OPTION_NONE_MISSING]: "No obvious questions missing",
-      },
     },
   },
   [FeedbackStage.OVERALL_ANS]: {
     [EvalType.RIG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_ANS_KEY,
+          question: "How is the overall answer?",
+          responseOptions: {
+            [QUERY_OVERALL_OPTION_HALLUCINATION]: "Found factual inaccuracies",
+            [QUERY_OVERALL_OPTION_OK]: "No obvious factual inaccuracies",
+          },
+        },
+      ],
       title: "OVERALL EVALUATION",
-      question: "How is the overall answer?",
-      responseOptions: {
-        [QUERY_OVERALL_OPTION_HALLUCINATION]: "Found factual inaccuracies",
-        [QUERY_OVERALL_OPTION_OK]: "No obvious factual inaccuracies",
-      },
     },
     [EvalType.RAG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_ANS_KEY,
+          question: "How relevant is the overall answer to the query?",
+          responseOptions: {
+            [QUERY_OVERALL_OPTION_IRRELEVANT]: "Not at all relevant",
+            [QUERY_OVERALL_OPTION_SOMEWHAT_RELEVANT]: "Somewhat relevant",
+            [QUERY_OVERALL_OPTION_RELEVANT]: "Relevant",
+          },
+        },
+        {
+          firestoreKey: QUERY_OVERALL_ANS_ACCURACY_KEY,
+          question: "What is the quality of the answer?",
+          responseOptions: {
+            [QUERY_OVERALL_OPTION_HALLUCINATION]:
+              "Poor narrative and/or with hallucinations",
```

Comment:
Trying to think if we should distinguish this from the relevant question.  Otherwise it feels confusing. @shifucun

What do you think of:

For relevant answers, how is the overall answer quality?
- Bad (Poor narrative, no stats references, etc)
- Average
- Good (Well written answer with appropriate statistics)

Reply:
to clarify, this is an overall evaluation without other conditions?

Reply:
hmm, is it though - i.e., do we need a further classification for an answer that is irrelevant?

Reply:
updated according to offline discussion